impedance/circuits.py

```diff
@@ -55,7 +55,7 @@ def __eq__(self, other):
         else:
             raise TypeError('Comparing object is not of the same type.')
 
-    def fit(self, frequencies, impedance, method='lm', bounds=None):
+    def fit(self, frequencies, impedance, method=None, bounds=None):
```

Comment:
I think we should leave the default as lm (that way we can still just call .fit(f, Z) without having to pass a method every time)

Reply:
Leaving lm as default actually breaks things, I think. If None is passed to method scipy will default to trf/lm for a function call with/without bounds.

Reply:
Ah, gotcha :+1: